doc/arch/adr-005-api-client-app-auth.md

```diff
@@ -0,0 +1,121 @@
+# Geoprocessing API: Only Allow Anonymous Requests From The Client App
+
+## Context
+
+We're creating a [proper, publicly documented API](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/geoprocessing_api/views.py) around our analyze and 
+RWD endpoints: the [Geoprocessing API.](https://github.com/WikiWatershed/model-my-watershed/blob/develop/doc/arch/adr-004-geoprocessing-api.md) The analyze and RWD endpoints have
+have allow-any access; we'd now like to secure them via token authentication
+so that we can better track individual users, prevent users from sending off too many
+requests, revoke the access of problematic users, and some day maybe add a 
+paid tier of use.
+
+Enforcing token authentication on these previously open endpoints causes
+a problem for our client app; the client app consumes the same API, and
+we want to allow it to use these endpoints un-authenticated. We need a way 
```

Comment:
Consider rephrasing to illustrate that the endpoints may be authenticated, but unauthenticated users of the app should continue to be able to access them.

Something like "..want to continue to allow unauthenticated users access to these endpoints via the app"